FhotoFithFriends/pom.xml

```diff
@@ -21,6 +21,14 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter</artifactId>
 		</dependency>
+		<dependency>
```

Comment:
Added thymeleaf dependency and spring starter web dependency for application controller